packages/inscription-view/src/components/parts/common/permission/Permission.test.tsx

```diff
@@ -2,14 +2,14 @@ import { CollapsableUtil, SelectUtil, render, screen } from 'test-utils';
 import { Permission } from './Permission';
 import { describe, test, expect } from 'vitest';
 
-const defaultConfig = { anonymous: true, error: 'Everybody', role: 'ivy:security:forbidden' };
+const defaultConfig = { anonymous: true, error: 'Everybody', roles: ['ivy:security:forbidden'] };
```

Comment:
arn't here the things switched?
the error should be 'ivy:security...' and the roles should be everybody?